Gemfile

```diff
@@ -58,8 +58,8 @@ gem 'rest-client'
 
 gem 'noid', '~> 0.8'
 
-# pin this to post-USN-3271-1
-gem 'nokogiri', '~> 1.7.2'
+# pin this to post-USN-3424-1
+gem 'nokogiri', '~> 1.8.1'
```

Comment:
Somehow ironic that XML is our number one source of security-related upgrades

Reply:
XML is pretty crazy though: https://blog.pragmatists.com/xml-be-cautious-69a981fdc56a